packages/button/src/constants/buttonVariants.js

```diff
@@ -1,11 +1,8 @@
-const BUTTON_TYPES = {
+export default {
   PRIMARY: "PRIMARY",
   NEUTRAL: "NEUTRAL",
   SECONDARY: "SECONDARY",
   NEGATIVE: "NEGATIVE",
-  CLOSED: "CLOSED",
   POSITIVE: "POSITIVE",
-  OFF_NEUTRAL: "OFF_NEUTRAL",
+  GHOST: "GHOST",
 };
```

Comment:
Since we're having breaking changes, how about adding a namespace to the constants?
```
  PRIMARY: "PRIMARAY_BUTTON",
  NEUTRAL: "NEUTRAL_BUTTON",
  . . .
```

In case we decide to use the same theming structure to another component that would have similar variants  (like `Link`), the settings could clash if we use only the variant name.

Reply:
Great suggestion!